modules/dag/src/main/scala/org/tessellation/dag/l1/ConsensusConfig.scala

```diff
@@ -0,0 +1,4 @@
+package org.tessellation.dag.l1
+
+// TODO: consider using posInt
+case class ConsensusConfig(peersCount: Int = 2, tipsCount: Int = 2)
```

Comment:
It's a part of `DAGStateChannelContext` which is created during startup, so this config taken from context is not intended to change during runtime. Though it could change during state channel startup.